pynest/nest/spatial/hl_api_spatial.py

```diff
@@ -46,6 +46,8 @@ class DistanceParameter(Parameter):
     be used in contexts with two nodes, e.g. when connecting.
     """
 
+    __distanceParam = None
```

Comment:
I think we only need a single underscore here instead of the double one.

Reply:
I have tested it with double underscores, and it makes the `__distanceParam` not accessible from outside the class.

Reply:
name wrangling with double underscore isn't usually recommended, unless you have a complex hierarchy where you know each class in the hierarchy will need to lookup their own possibly overlapping class attributes.

> and it makes the __distanceParam not accessible from outside the class.

This is not true, and why it's not recommended: access modifiers don't exist in Python, your attribute is still accessible as `f"_{self.__class__.__name__}_distanceParam"` ;p